vms/platformvm/validators/manager.go

```diff
@@ -131,7 +179,12 @@ func (m *manager) GetMinimumHeight(ctx context.Context) (uint64, error) {
 	return blk.Height() - 1, nil
 }
 
-func (m *manager) GetCurrentHeight(context.Context) (uint64, error) {
+func (m *manager) GetCurrentHeight(ctx context.Context) (uint64, error) {
+	return m.getCurrentHeight(ctx)
+}
```

Comment:
This change is a bit forward looking (as with the caching changes) to better enable concurrent access.